eZ/Publish/API/Repository/TransactionService.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Publish\API\Repository;
```

Comment:
```suggestion
declare(strict_types=1);

namespace eZ\Publish\API\Repository;
```

Reply:
Fixed in https://github.com/ezsystems/ezplatform-kernel/pull/164/commits/ffb35ff9c0bd6d94b14a1401cfb71a4e9312cb44

Reply:
+1 for comments/strings